logstash-core/src/main/java/org/logstash/Timestamp.java

```diff
@@ -12,10 +12,10 @@
 import org.logstash.json.TimestampSerializer;
 
 @JsonSerialize(using = TimestampSerializer.class)
-public final class Timestamp implements Cloneable, Comparable<Timestamp>, Queueable {
+public final class Timestamp implements Comparable<Timestamp>, Queueable {
```

Comment:
@jakelandis fair point, actually there are some sensible things to add here, will do in a bit.